server/tests/src/test/java/org/infinispan/server/functional/rest/RestServerResource.java

```diff
@@ -93,4 +93,15 @@ public void testCacheManagerNames() {
       Set<String> cmNames = cacheManagers.asJsonList().stream().map(Json::asString).collect(Collectors.toSet());
       assertEquals(cmNames, Sets.newHashSet("default"));
    }
+
+   @Test
+   public void testCacheDefaults() {
```

Comment:
I believe it is worth validating more fields here. And another good test to have would be to retrieve the defaults and create a new cache with the return.